### PR TITLE
Translate some lighthouse audits into korean

### PR DESCRIPTION
**Pull request by techhtml**
* common query
* common implementation-heading
* aria-allowed-attributes
* content-sized-correctly-for-viewport
* critical-request-chains
* valid-aria-attributes

### CODE REVIEW

File: src/content/ko/tools/lighthouse/audits/_shared/query.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{# wf_md_include #}
+
+아래 검사에서, Lighthouse는 `1 element fails this test` 같은 걸 보여줍니다.
+번호는 실패한 요소의 갯수에 따라 다릅니다. 목록을 확장하려면 이 레이블을 클릭하세요.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/_shared/query.md]: 번호 -> 숫자

File: src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+잘못된 role-속성 조합은 페이지의 접근성을 불가하게 할 수 있습니다.
+
+## 어떻게 이 검사를 통과하는가{: #how }
+
+<<_shared/query.md>>
+
+각 요소의 잘못된 조합(들)을 찾으려면 다음을 수행하세요:
+
+1. 요소의 `role` 및 `aria-*` 속성을 보십시오.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md]: 보십시오 -> 확인하세요

File: src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+1. [Definition of Roles][roles]로 이동하세요.
+
+1. 이 요소의 `role` 페이지로 이동하세요.
+
+1. **Required States and Properties**나 **Supported States and Properties** 목록에서
+  요소의 `aria-*` 속성을 다시한번 체크하세요. 이 두 목록 중 하나에없는 속성은 유효하지 않습니다.
+
+유효하지 않은 콤보를 수정하려면 요소에서 유효하지 않은 속성을
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md]: 콤보 -> 조합

File: src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Element ARIA Attributes Are Allowed For This Role" Lighthouse audit.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md]: description도 번역하시기를 추천합니다.

File: src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Content Sized Correctly for Viewport" Lighthouse audit.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md]: description도 번역하시기를 추천합니다.

File: src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+{# wf_updated_on: 2017-02-12 #}
+{# wf_published_on: 2016-10-04 #}
+
+# 콘텐츠 사이즈를 뷰포트에 맞춘다  {: .page-title }
+
+## 왜 이 검사가 중요한가 {: #why }
+
+이 검사는 페이지 콘텐츠 너비가 뷰포트 너비와 동일한 지 검사합니다.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md]: 동일한 지 -> 동일한지

> [막연한 의문이 있는 채로 그것을 뒤 절의 사실이나 판단과 관련시키는 데 쓰는 연결 어미는 붙여씁니다.](https://twitter.com/urimal365/status/464302369958670338)

File: src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+콘텐츠 너비가 뷰포트 너비보다 작거나 큰 경우 페이지가 모바일 화면 용으로 최적화되어 있지 않은 경우가 종종 있습니다.
+
+## 어떻게 이 검사를 통과하는가 {: #how }
+
+이 검사는 페이지가 모바일 디바이스에 맞게 최적화되어있는 지 우회해서 확인하는 방법입니다.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md]: 있는 지 -> 있는지

File: src/content/ko/tools/lighthouse/audits/critical-request-chains.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Critical Request Chains" Lighthouse audit.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/critical-request-chains.md]: description 번역 :)

File: src/content/ko/tools/lighthouse/audits/valid-aria-attributes.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+제대로 이해하지 못할 수 있습니다. 이로 인해 스크린 리더에 의존하는 사용자가
+페이지를 사용하지 못할 수 있습니다.
+
+## 어떻게 이 검사를 통과하는가 {: #how }
+
+<<_shared/query.md>>
+
+리스트의 각 요소에서 잘못된 속성명을 찾으려면 다음을 수행하세요:
+
+1. 요소의 `role` 및 `aria-*` 속성을 보십시오.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/audits/valid-aria-attributes.md]: 보십시오 -> 확인하세요.

**Comment by cwdoh** [comment]
Oops. I mistyped emoticon. :)

**Comment by cwdoh** [comment]
@techhtml please, update all `wf_updated_on` tags. :)

```
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/aria-allowed-attributes.md#5 WF Tag `wf_updated_on` must be within the last 7 days.
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md#5 WF Tag `wf_updated_on` must be within the last 7 days.
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/content-sized-correctly-for-viewport.md#18 Line must be at most 100 characters
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/critical-request-chains.md#5 WF Tag `wf_updated_on` must be within the last 7 days.
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/critical-request-chains.md#60 Line must be at most 100 characters
[15:44:46] ERROR: src/content/ko/tools/lighthouse/audits/valid-aria-attributes.md#5 WF Tag `wf_updated_on` must be within the last 7 days.
```

**Comment by techhtml** [comment]
@cwdoh i fixed that.

**Comment by cwdoh** [comment]
Ok, it is good to go. :)

**Comment by techhtml** [comment]
@petele Please merge this

**Comment by agektmr** [comment]
I will merge this translation but needs to resolve the conflicts. Can @techhtml or @cwdoh help?

**Comment by techhtml** [comment]
I tried fix it :) Thanks.

**Comment by agektmr** [comment]
@techhtml I still see there're
screen shot 2017-11-24 at 15 41 00
 conflicts?

**Comment by cwdoh** [comment]
still LGTM, but there're slight changes from original articles such as adding references. @techhtml Would you check them, please? :)

**Comment by techhtml** [comment]
Sorry for rate modify. I fixed conflicts :)

**Comment by techhtml** [comment]
Sorry for rate modify. I fixed conflicts :)

**Comment by cwdoh** [comment]
@techhtml I recommends '검사' rather than '감사'. 검사 looks more natural to be understood. :)

**Pull request merged**